src/jdk.jdwp.agent/share/native/libjdwp/threadControl.c

```diff
@@ -2623,15 +2623,58 @@ dumpThreadList(ThreadList *list)
     }
 }
 
+#ifdef DEBUG_THREADNAME
+static void
+setThreadName(ThreadNode *node)
+{
+  /*
+   * Sometimes the debuggee changes the thread name, so we need to fetch
+   * and set it again.
+   */
+  jvmtiThreadInfo info;
+  jvmtiError error;
+
+  memset(&info, 0, sizeof(info));
+  error = JVMTI_FUNC_PTR(gdata->jvmti,GetThreadInfo)
+    (gdata->jvmti, node->thread, &info);
+  if (info.name != NULL) {
+    strncpy(node->name, info.name, sizeof(node->name) - 1);
+    jvmtiDeallocate(info.name);
+  }
+}
+#endif
+
+#if 0
+static jint
+getThreadState(ThreadNode *node)
+{
+  jint state = 0;
+  jvmtiError error = FUNC_PTR(gdata->jvmti,GetThreadState)
+    (gdata->jvmti, node->thread, &state);
+  return state;
```

Comment:
Nit: The indent in this file is 4, not 2.

Reply:
Fixed.